conf/local.conf.sample

```diff
@@ -12,10 +12,24 @@
 # variable as required.
 
 #
-# Machine Selection
+# Generic Machine Selection
 #
-# You need to select a specific machine to target the build with. There are a selection
-# of machines available which can boot and run BISDN Linux.
+# You need to select a specific machine to target the build with. Each machine
+# targets multiple switch platforms.
```

Comment:
I found the multiple uses of "target" here a bit confusing. Also, the "specific machine" here refers to the yocto MACHINE which can be a totally unspecific generic-x86-64. The user can figure this out, but maybe we can help them a bit.

```suggestion
# You need to set the "machine" to be built by yocto. Each machine
# can support multiple switch platforms.
```

Reply:
The first sentence is verbatim taken from the default yocto [local.conf.sample](https://git.yoctoproject.org/poky/tree/meta-poky/conf/templates/default/local.conf.sample#n16). 

Reworded the second sentence, and reworded the description of where to find the list of platforms to avoid reuse of machine.